### DISCUSSION TITLE
Welcome to unicodify Discussions!

### DISCUSSION THREAD

**Discussion by rugk** [rugk/unicodify#31] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Thomas-D-C** [comment]
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Hi :-)

When I type texts I'm lazy, and the 3 most important features I need are:
- spell checker -> built-in,
- case handling -> Unicodify,
- shortcut substitution -> [Text Insert](https://addons.thunderbird.net/fr/thunderbird/addon/text-insert-text-blocks/) (partially).


I'm glad this can work in Firefox forms! :-)
I didn't imagine it was possible, and I stayed for about 6 months having installed it only in Thunderbird! :-/
